tests/client/amo/sagas/testUtils.js

```diff
@@ -0,0 +1,11 @@
+import { getApi } from 'amo/sagas/utils';
+import { signedInApiState } from 'tests/client/amo/helpers';
+
+
+describe('Saga utils', () => {
+  it('should return API state', () => {
+    const state = { api: signedInApiState, somethingElse: true };
+
+    assert.deepEqual(getApi(state), signedInApiState);
```

Comment:
This would be a better test if it used the `SagaTester`. I don't know how hard that is but the `redux-saga-tester` docs have some good examples of adding a saga function to the tester.